pkg/cloud/prepare/rhos.go

```diff
@@ -38,21 +38,21 @@ func RHOS(clusterInfo *cluster.Info, ports *cloud.Ports, config *rhos.Config, us
 	//nolint:wrapcheck // No need to wrap errors here.
 	err = rhos.RunOn(clusterInfo, config, status,
 		func(cloud api.Cloud, gwDeployer api.GatewayDeployer, status reporter.Interface) error {
+			if len(internalPorts) > 0 {
```

Comment:
In case of OVN the internalPorts will be empty right. So how does this fix the issue?

Reply:
This PR just create internal SG before g//w node is deployed. This along with https://github.com/submariner-io/cloud-prepare/pull/671/files fixes the issue.